src/forms/add-template.schema.js

```diff
@@ -5,7 +5,7 @@ import validatorTypes from '@data-driven-forms/react-form-renderer/validator-typ
 import loadNotificationSettingsOptions from './load-notification_settings-debounced';
 import templateMessages from '../messages/templates.messages';
 
-const addTemplateSchema = (intl, id = []) => {
+const addTemplateSchema = (intl, id) => {
```

Comment:
Hmm maybe I'm thinking of a different file, but didn't this `id= []` just get added in #108 ? Was it just not necessary?